dbms/src/Storages/Transaction/ApplySnapshot.cpp

```diff
@@ -337,6 +337,9 @@ std::vector<DM::ExternalDTFileInfo> KVStore::preHandleSSTsToDTFiles(
 
     std::vector<DM::ExternalDTFileInfo> generated_ingest_ids;
     TableID physical_table_id = InvalidTableID;
+
+    auto region_id = new_region->id();
+    auto prehandle_task = prehandling_trace.registerTask(region_id);
```

Comment:
can we check whether `prehandle_task` is `true` or not after the following lines? If it is true, then we can use `stream->cancel` to remove the generated files instead of using another FFI function `ReleasePreHandledSnapshot`
 https://github.com/pingcap/tiflash/blob/e922e63ac9567f4b54527c12676252242f8ab8c7/dbms/src/Storages/Transaction/ApplySnapshot.cpp#L397-L400

Reply:
I will fix that